<launch>
    <node pkg="nate_stretch_movement" type="turn_left_90_node" name="Left_Turn_Node" output="screen">
        <rosparam file="$(find nate_stretch_movement)/params/right_wall_hugger_params.yaml" command="load"/>
    </node>
</launch>